infra/modules/aws-psoxy-lambda/main.tf

```diff
@@ -70,7 +70,25 @@ resource "aws_iam_role" "iam_for_lambda" {
   }
 }
 
-resource "aws_iam_policy" "policy" {
+# "PSOXY_${upper(replace(each.value.connector_name, "-", "_"))}_${upper(each.value.secret_name)}"
+data "aws_ssm_parameters_by_path" "psoxy_parameters" {
+  path            = "/"
+  with_decryption = false # we just want the arns, not the values
+}
+
+locals {
+  prefix          = "PSOXY_${upper(replace(var.source_kind, "-", "_"))}_"
+  prefix_writable = "PSOXY_${upper(replace(var.source_kind, "-", "_"))}_WRITABLE"
```

Comment:
WRITABLE parameters by convention.
This block reads all parameters and filters by prefix to grant read/write to specific parameters of the function.
The external ones needs to be passed in `parameters`

Reply:
changed for an object with a writeable attribute